packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1122,6 +1203,7 @@ export default class Home extends Component {
                 )}
                 <DropdownButton
                   bsStyle='link'
+                  disabled={disableAction}
```

Comment:
So it should probably not be called `disableAction`. Maybe call it `disableHomeFeatures` and add a comment when you declare it explaining why it's necessary since all the features we disable are either handled by the SortedTable or irrelevant with the SortedTable.